_unittest/test_01_Design.py

```diff
@@ -204,14 +204,21 @@ def test_19_import_dataset1d(self):
         assert self.aedtapp.import_dataset1d(filename)
 
     def test_19_import_dataset3d(self):
-        filename = os.path.join(local_path, "example_models", "ds_3d.tab")
+        filename = os.path.join(local_path, "example_models", "Dataset_3D.tab")
         ds8 = self.aedtapp.import_dataset3d(filename)
-        assert ds8.name == "$ds_3d"
-        ds9 = self.aedtapp.import_dataset3d(filename, dsname="dataset_test")
-        assert ds9.name == "$dataset_test"
-        assert ds9.delete()
+        assert ds8.name == "$Dataset_3D"
+        filename = os.path.join(local_path, "example_models", "Dataset_3D.csv")
+        ds8 = self.aedtapp.import_dataset3d(filename, dsname="dataset_csv")
+        assert ds8.name == "$dataset_csv"
+        assert ds8.delete()
         ds10 = self.aedtapp.import_dataset3d(filename, dsname="$dataset_test")
-        assert ds10.vunit == "m_per_sec"
+        assert ds10.zunit == "mm"
+        filename = os.path.join(local_path, "example_models", "Dataset_3D.csv")
+        ds8 = self.aedtapp.import_dataset3d(filename, encoding="utf-8-sig", dsname="dataset_csv")
+        assert ds8.name == "$dataset_csv"
+        filename = os.path.join(local_path, "example_models", "Dataset_3D.xlsx")
```

Comment:
To test the excel import you need pandas on the build machine. add it to requirements_test.txt